### PR TITLE
Fix slimeboots & slimesling

### PR DESCRIPTION
**Pull request by mitchej123**
* Use vec3 instead of Vector3d which isn't found on the server
* set entity.motionXYZ manually, as setVelocity is ClientOnly
  * Close https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11140

### CODE REVIEW

File: src/main/java/tconstruct/gadgets/TinkerGadgets.java [hunk 1]
```diff
@@ -32,14 +32,15 @@ public class TinkerGadgets {
 
     @Handler
     public void preInit(FMLPreInitializationEvent event) {
-
+        log.info("Pre Init");
```
Comment (querns) [inline src/main/java/tconstruct/gadgets/TinkerGadgets.java:35]: Missed removing a logging statement.

Comment (mitchej123) [reply src/main/java/tconstruct/gadgets/TinkerGadgets.java:35]: I liked them.  Can make them more witty ;)

**Review by minecraft7771 — APPROVED** [review]

**Comment by ORanGeOfficial** [comment]
> * Use vec3 instead of Vector3d which isn't found on the server
> * set entity.motionXYZ manually, as setVelocity is ClientOnly
>   
>   * Close [Crash with tinkers construct slime boots GT-New-Horizons-Modpack#11140](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11140)

Can you please add a recipe for slimeboots and slimesling? I was so happy to see such a cool mechanic was backported, but then I discovered there are no recipes for the items.

**Comment by Dream-Master** [comment]
> > * Use vec3 instead of Vector3d which isn't found on the server
> > * set entity.motionXYZ manually, as setVelocity is ClientOnly
> >   
> >   * Close [Crash with tinkers construct slime boots GT-New-Horizons-Modpack#11140](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11140)
> 
> Can you please add a recipe for slimeboots and slimesling? I was so happy to see such a cool mechanic was backported, but then I discovered there are no recipes for the items.

you can find then only but if you have an idea for a recipe why not

**Pull request merged**